packages/codemods/src/codeMods/ICodeMod.ts

```diff
@@ -0,0 +1,28 @@
+import { SourceFile } from 'ts-morph';
+
+export interface ICodeModResult {
```

Comment:
New packages should probably omit the I from interfaces. create-package templates ought to be setting it up that way automatically.